.github/workflows/reusable.rebuild.yml

```diff
@@ -252,14 +256,17 @@ jobs:
     - name: Build LLVM
       shell: cmd
       run: |
-        python ./alloy.py -b --version=${{ inputs.version }} --verbose --generator="${{ inputs.vs_generator }}"
-        cd alloy_results* && type alloy_build.log
+        mkdir build-${{ inputs.version }}
+        cd build-${{ inputs.version }}
+        cmake %ISPC_HOME%\superbuild --preset os -G "NMake Makefiles" -DLLVM_VERSION=${{ inputs.version }} -DCMAKE_INSTALL_PREFIX=%LLVM_HOME%\bin-${{ inputs.version }} -DBUILD_STAGE2_TOOLCHAIN_ONLY=ON -DXE_DEPS=OFF -DLLVM_DISABLE_ASSERTIONS=OFF
+        nmake
+        cd ../
+        rmdir /s /q build-${{ inputs.version }}
```

Comment:
I'm wondering why is it needed on the shared runner? Are we running out of disk space?

Reply:
Yes, [here](https://github.com/nurmukhametov/ispc/actions/runs/5610110736) it happened.